docs/_data/jekyll_variables.yml

```diff
@@ -120,6 +120,13 @@ page:
       <code>/work/code/_posts/2008-12-24-closures.md</code> would have this field set to
       <code>['work', 'code']</code>. These can also be specified in the
       <a href="/docs/front-matter/">front matter</a>.
+   - name: page.collection
+    description: >-
```

Comment:
The indentation of `description` key is incorrect.
```yaml
- name: page.collection
  description: >-
    The the collection to which this page belongs. This would equal the string `posts`
```